engine/dlib/src/dlib/log.cpp

```diff
@@ -458,9 +478,22 @@ void UnregisterLogListener(LogListener listener)
     dmLogWarning("dmLog listener not found");
 }
 
-#undef MAX_LISTENERS
+#undef g_dmLog_MaxListeners
 
-void LogInternal(Severity severity, const char* domain, const char* format, ...)
+void dmLogSetLevel(LogSeverity severity)
+{
+    dmLog::g_LogLevel = severity;
+}
+
+// Deprecated. Try to move back to the C api
+namespace dmLog {
+    void RegisterLogListener(FLogListener listener)     { dmLogRegisterListener(listener); }
+    void UnregisterLogListener(FLogListener listener)   { dmLogUnregisterListener(listener); }
+    void Setlevel(LogSeverity severity)                 { dmLogSetLevel(severity); }
+}
```

Comment:
We should communicate this to our users

Reply:
Could you please add a comment here (and assign this Pr to the next version)
<img width="929" alt="image" src="https://user-images.githubusercontent.com/2209596/179393004-4ed8c853-5b7d-48a1-af00-83ad6be1846e.png">

Then this text will be taken by script into release logs